media/pages/sk/podporte-nas/page.html

```diff
@@ -0,0 +1,26 @@
+{# vim: set filetype=htmldjango shiftwidth=2 :#}
+{% load page from poleno.pages %}
+
+
+<h1>Podporte nás</h1>
+<p>
+Názov združenia: Chcemvediet.sk
+<br/>
+Sídlo: Karpatské námestie 10A, 83106 Bratislava-Rača, Slovenská republika
+<br/>
+IČO: 50436376
+</p>
```

Comment:
Pls odsadzuj vnorene tagy.

Text pred dvojbodkou oznac `<strong>`, nech sa to lepsie cita.